### PR TITLE
fix: should use port 3000 in agentportal svc

### PR DESCRIPTION
**Pull request by bjwswang**
#### Which issue(s) this PR fixes

Fixes #

### CODE REVIEW

File: deploy/charts/arcadia/templates/agent-portal.yaml [hunk 1]
```diff
@@ -41,6 +41,8 @@ spec:
         ports:
         - containerPort: 80
```
Comment (Carrotzpc) [inline deploy/charts/arcadia/templates/agent-portal.yaml:42]: deploy 中只需要暴露 3000 端口即可

File: deploy/charts/arcadia/templates/agent-portal.yaml [hunk 2]
```diff
@@ -41,6 +41,8 @@ spec:
...
           protocol: TCP
```
Comment (nkwangleiGIT) [inline deploy/charts/arcadia/templates/agent-portal.yaml:43]: we can remove 80 port if it's not used

File: deploy/charts/arcadia/templates/agent-portal.yaml [hunk 3]
```diff
@@ -76,7 +78,7 @@ spec:
   internalTrafficPolicy: Cluster
   ports:
   - name: http
-    port: 3000
+    port: 80
     protocol: TCP
```
Comment (nkwangleiGIT) [inline deploy/charts/arcadia/templates/agent-portal.yaml:82]: or we can use both 3000 for port and targetPort, and use 3000 directly in the ingress port number

File: deploy/charts/arcadia/templates/agent-portal.yaml (hunk 2 above)
Comment (bjwswang) [reply deploy/charts/arcadia/templates/agent-portal.yaml]: 3000

**Comment by bjwswang** [comment]
@nkwangleiGIT @Carrotzpc  All changed to 3000 .

**Pull request merged**